README.md

```diff
@@ -186,3 +190,7 @@ To handle and display rdf-files, especially .ttl files, you can use an IDE with
       }
   ```
   This is obviously not conform since the mandatory files `url` and `type` of `LinkShape` are missing. This bug will be fixed in the future.
+
+### Issues with Signature Validation in [Federated Catalogue](fc-server.gxfs.gx4fm.org)
```

Comment:
Hab mir kurz angeschaut, woran das liegt.
Ich würde es nicht als Issue mit Signature Validation aufhängen, sondern an Issue with SD-Creation-Wizard, also diese Liste erweitern mit dem neuen Punkt.

Problem macht die Range2DShape, die in der general-SHACL parallel zur GeneralShape definiert ist und in der HDMap eingebunden wird.
Daraus macht der Wizard eine unsinnige jsonld Repräsentation.
Hinweis wäre damit, dass wenn man eine "externe" Shape, wie z. B. Range2DShape einbindet, man prüfen soll, ob im instance-file diese richtig eingehangen wurde und nicht doppelt ist. Dazu sollten wir direkt noch einen Issue im SD-Creation-Wizard anlegen und hier verlinken. 

Damit wird glaub ich der "### Instance"-Abschnitt obsolet, d. h. wir können den erstmal entfernen aus meiner Sicht.